packages/react/src/account/useAccountManagers.ts

```diff
@@ -0,0 +1,42 @@
+import type { AccountManager, AccountManagersRequest, Paginated } from '@lens-protocol/graphql';
+import { AccountManagersQuery } from '@lens-protocol/graphql';
+
+import type { ReadResult, Suspendable, SuspendableResult, SuspenseResult } from '../helpers';
+import { useSuspendableQuery } from '../helpers';
+
+export type AccountManagersArgs = AccountManagersRequest;
+
+/**
+ * Fetch Account Managers.
+ *
+ * This signature supports React Suspense:
+ *
+ * ```tsx
+ * const { data } = useAccountManagers({ suspense: true });
+ * ```
+ */
+export function useAccountManagers(
+  args: AccountManagersArgs & Suspendable,
+): SuspenseResult<Paginated<AccountManager>>;
+
+/**
+ * Fetch Account Managers.
+ *
+ * ```tsx
+ * const { data, loading } = useAccountManagers();
+ * ```
+ */
+export function useAccountManagers(
+  args: AccountManagersArgs,
+): ReadResult<Paginated<AccountManager>>;
+
+export function useAccountManagers({
+  suspense = false,
+  ...request
+}: AccountManagersArgs & { suspense?: boolean }): SuspendableResult<Paginated<AccountManager>> {
```

Comment:
I believe in the case of optional args for non-suspense signature we need to also provide a default value here:

```suggestion
}: AccountManagersArgs & { suspense?: boolean } = {}): SuspendableResult<Paginated<AccountManager>> {
```

This is not visible from the outside, it's just needed to TS to support all signatures.